osa/configs/sequencer.cfg

```diff
@@ -137,6 +138,7 @@ WORKFLOWPREFIX: Workflow
 GRAPHSUFFIX: .dot
 SVGSUFFIX: .svg
 end_of_activity: NightFinished.txt
+gain_selection_chek: GainSelFinished.txt
```

Comment:
gain_selection_chek -> gain_selection_check
one of the tests is failing because of this